.pre-commit-config.yaml

```diff
@@ -103,6 +103,13 @@ repos:
             |\.\.\ code-block\ ::
             |\.\.\ ipython\ ::
         types_or: [python, cython, rst]
+    -   id: cython-casting
+        name: Cython Casting
+        language: pygrep
+        entry: |
+            # Cython casting is of the form `<type>obj` as opposed to `<type> obj`
+            |[a-zA-Z0-9*]>\s
```

Comment:
this can probably just fit on one line: `entry: [a-zA-Z0-9*]>\s`

Reply:
I tried that, but I got the following error.

<img width="632" alt="Screenshot 2021-08-02 at 10 00 55 PM" src="https://user-images.githubusercontent.com/46242526/127894291-c9f19d1f-6a3d-43c9-8c69-e6e43ebf6afe.png">

Reply:
ah you probably just need to quotes around the string

Reply:
Thank you it worked, will update the PR. Also, now I can just use the space at the end as before.